instaloader/__main__.py

```diff
@@ -111,6 +112,10 @@ def import_session(browser, instaloader, cookiefile):
         if not username:
             raise SystemExit(f"Not logged in. Are you logged in successfully in {browser}?")
         instaloader.context.username = username
+        print(f"{username} has been successfully logged in.")
+        next_step_text = (f"Next: Run instaloader --login={username} as it is required to download high quality media "
+            "and to make full use of instaloader's features.")
+        print(textwrap.fill(next_step_text, width=80))
```

Comment:
In another place we use `textwrap.fill()` without any explicit `width` parameter, leading to the default of 70. We should be consistent here, I do not see a need to have a wider text at this place.

Reply:
Remove the width parameter.